ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -362,6 +362,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec
+- as: azure-ipi-private-workers-marketimage-p2-f7
```

Comment:
By a chance, I found the job on line 341 is using a wrong workflow, can you help update the workflow to cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy together in this PR.

Reply:
If you do not mind, also need to update the same job in `openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13`, thanks in advance.

Reply:
done